pkgs/os-specific/linux/kernel/hardened/config.nix

```diff
@@ -78,6 +78,13 @@ assert (versionAtLeast version "4.9");
   PROC_KCORE         = no; # Exposes kernel text image layout
   INET_DIAG          = no; # Has been used for heap based attacks in the past
 
+  # INET_DIAG=n causes the following options to not exist anymore, but since they are defined in common-config.nix,
+  # make them optional
+  INET_DIAG_DESTROY = option no;
```

Comment:
Would `INET_DIAG_DESTROY = null` unset this option?

Reply:
`error: The option value `settings.INET_DIAG_DESTROY' in `structuredExtraConfig' is not of type `submodule'.`

Maybe this could be made to work by adjusting the type a bit, but I don't think there's much point investing any effort into that when we can leave it as `option no`.